evm/src/generation/mpt.rs

```diff
@@ -33,92 +33,96 @@ impl Default for AccountRlp {
     }
 }
 
-#[derive(RlpEncodable, RlpDecodable, Debug)]
-pub struct AccessListItemRlp {
-    pub address: Address,
-    pub storage_keys: Vec<U256>,
-}
+pub mod transaction_testing {
```

Comment:
tiny nit: Having this module in the middle of the file looks a bit weird. Maybe we can move it to the end of the file?